connexion/decorators/security.py

```diff
@@ -28,6 +29,20 @@
 session.mount('https://', adapter)
 
 
+def get_tokeninfo_url(security_definition):
+    '''
+    :type security_definition: dict
+    :rtype: str
+
+    >>> get_tokeninfo_url({'x-tokenInfoUrl': 'foo'})
+    'foo'
+    '''
+    token_info_url = security_definition.get('x-tokenInfoUrl') or \
+        os.environ.get('HTTP_TOKENINFO_URL') or \
```

Comment:
Maybe we should log a warning message notifying the user -- if the `HTTP_TOKENINFO_URL` is set -- that they should use `TOKENINFO_URL`?